### DISCUSSION TITLE
Issue with interaction with templater

### DISCUSSION THREAD

**Discussion by prototyperafa** [chhoumann/quickadd#596] [Q&A]
I have a template like the image below:
![image](https://github.com/chhoumann/quickadd/assets/42654326/ec820c02-3164-45fe-85b3-3b6217554ae1)

I'm trying to automate a process whick uses quickAdd to modify this template and add something below "Hábitos", however, there seems to be an interaction with the date line, because when I run quickadd it gives me this error

![image](https://github.com/chhoumann/quickadd/assets/42654326/fcd9d0b7-b3d3-41e0-ab4e-713bfda9253e)

I use this template along with daily notes and would like to keep the line, is there a way to make quickAdd "ignore" it?

Thanks in advance :)

**Comment by chhoumann** [comment]
Sorry for the very slow reply.

I can't read the exact error from the screenshots, but the pattern is most likely this: your daily note keeps a live Templater expression (the date line), and back when you posted this, QuickAdd ran Templater over the whole destination file after a capture - so that raw `<% ... %>` line got re-evaluated outside the context it expects and errored. On current QuickAdd that whole-file pass is opt-in (a Behavior toggle on the Capture choice called "Run Templater on entire destination file after capture", off by default), so the clash you hit shouldn't occur anymore unless that toggle is on.

If you want the note immune regardless of which plugin touches it, let QuickAdd own the daily note and use its own date token: on the Capture choice enable **Create file if it doesn't exist** with your template, and in that template replace the Templater date line with e.g. `{{DATE:DD-MM-YYYY}}` (any moment.js format works). QuickAdd renders it to plain text when the note is created, so later captures have nothing left to re-execute.

If you're on a current version and still hit this, paste the error as text from the developer console (Ctrl+Shift+I) rather than a screenshot and I'll take a look.
